### ISSUE TITLE
 isLockedOut() in Member.php call LoginAttempt::getByEmail but it passes to it the unique_identifier_field instead $this->Email

### ISSUE THREAD

**Issue by alessandromarotta** [silverstripe/silverstripe-framework#9507]
## Affected Version

Silverstripe framework ^4

## Description

Hello! the public function isLockedOut() in Member.php call LoginAttempt::getByEmail but it passes to it the unique_identifier_field value and this doesn't make sense for me: it could pass directly `$this->Email`.

This update could allow specific extension to patch the use of unique_identifier_field (otherwise it would be necessary to extends the Member Class to override the isLockedOut function, with a lot of problems)

Please, can you update that?

```diff
class Member extends DataObject
{

/**
     * Returns true if this user is locked out
     *
     * @skipUpgrade
     * @return bool
     */
    public function isLockedOut()
    {
        [...]
-        $idField = static::config()->get('unique_identifier_field');
-        $attempts = LoginAttempt::getByEmail($this->{$idField})
+        $attempts = LoginAttempt::getByEmail($this->Email)
            ->sort('Created', 'DESC')
            ->limit($maxAttempts);

       [...]
   }
}
```

Thank you

**Comment by dhensby** [comment]
Yep - I agree this should send the `Email` and not the `unique_identifier_field` - would you like to open a PR for it?

**Comment by alessandromarotta** [comment]
Yes, sure, thank you.

**Comment by robbieaverill** [comment]
Resolved in #9516
